spec.bs

```diff
@@ -384,30 +395,43 @@ try {
 }
 </pre>
 
-# Dependencies # {#sctn-dependencies}
+# Terminology # {#sctn-terminology}
+
+: <dfn export>SPC Credential</dfn>
+:: A WebAuthn credential that can be used for the behaviors defined in this specification.
+    Can be used as a normal credential (e.g. for authentication flows), as well
+    as for SPC behavior. A [=Relying Party=] must explicitly opt-in to making an
+    SPC credential during credential creation.
+
+: <dfn>Steps to determine if a credential is SPC-enabled</dfn>
+:: An as-yet undefined process by which a user agent can, given a [=credential
+    ID=], determine if the credential represented by that ID is an [=SPC
+    Credential=].
 
-This specification relies on several other underlying specifications, listed
-below and in [[#index-defined-elsewhere]].
+    This allows the user agent to ensure that the powers of an SPC credential 
+    are only applied to credentials created as SPC-enabled.
 
-: <dfn>WebAuthn Conditional UI</dfn>
-:: Secure Payment Confirmation only shows the transaction UX if one of the
-    passed credentials is valid for the current device, without requiring a user
-    interaction. This concept is not currently part of [[webauthn-3]], but is on
-    the roadmap for a future enhancement.
+    NOTE: See <a href="https://github.com/w3c/webauthn/issues/1667">WebAuthn
+    issue 1667</a>.
 
-    Until this is available, user agents can consider either storing a local
-    cache of WebAuthn credentials created for this device, or always showing the
-    transaction UX even if the user may be unable to complete the
-    authentication.
+: <dfn>Steps to silently determine if a credential is valid for the current device</dfn>
+:: An as-yet undefined process by which a user agent can, given a [=credential
+    ID=], silently (i.e., without user interaction) determine if the credential
+    represented by that credential ID is valid for the current device (i.e.,
```

Comment:
Can we use a term other than "valid"?  What about "is available" ?

Reply:
Done.